### ISSUE TITLE
Label is not cleared when input is cleared programmatically

### ISSUE THREAD

**Issue by Frank3K** [chemerisuk/better-dateinput-polyfill#87]
The `<time>`-label is not cleared when the `input[type="date"]` is cleared programmatically.

Reproduction scenario (tested on Chromium 53.0.2785.143):

1. Open the [demo](https://chemerisuk.github.io/better-dateinput-polyfill/).
2. Open the browser console and enter:

``` js
DOM.find('[data-polyfill="desktop"]').value("").fire("change")
```

Result: the label is not cleared.

**Comment by chemerisuk** [comment]
@Frank3K already fixed in master. I'll publish a bugfix version soon

**Comment by chemerisuk** [comment]
The fix was published in 2.0.1
